app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,75 @@
+
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
+// recommended for single sections: auto
+// recommended for longer questions: 50%
+
 .check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+  .set {
+    position: relative;
+    border-bottom: 1px solid $border-colour;
```

Comment:
I thought of using `>` again, which would make them more robust as well. But you wouldn't like that, right?
Do we have any specific namespace that we use? On the one hand we have `govuk-*`, but on the other hand that in itself might not specific enough and, confusingly, we're not using it consistently.
I gave the SASS variable a prefix of `cya` (short for 'check your answers'). Would that be good? Or a combination (`govuk-cya-*`)?